.github/workflows/rust.yml

```diff
@@ -133,6 +132,26 @@ jobs:
           fi
         shell: bash
 
+      - name: Download and install libtorch
+        run: |
+          import requests
+          import zipfile
+          import os
+      
+          version = '2.3.0' # default version
+          with open('requirements.txt') as f:
+              for line in f:
+                  if 'torch==' in line:
+                      version = line.split('==')[1].strip()
+                      break
+          response = requests.get(f'https://download.pytorch.org/libtorch/cpu/libtorch-macos-arm64-{version}.zip')
```

Comment:
I think an arm64 version of 2.3.0 torch exists as a Python wheel but if you need to download the zip. 
This one worked for me for (x86 at least):  
`wget https://download.pytorch.org/libtorch/cpu/libtorch-shared-with-deps-2.3.0%2Bcpu.zip`

Reply:
@AmineDiro Still being a pain 😡

Reply:
😔 i cant see the runner's error. Anything of help ?

Reply:
![image](https://github.com/huggingface/ratchet/assets/45471420/5299a469-1997-43c6-a8cb-ef42b1491b5a)

Seems tricky, I can look into it later 🫡

Reply:
Ohh I see that's the DYLIB export 👍🏼